src/matching/SubAST_generic.ml

```diff
@@ -302,13 +302,18 @@ let substmts_of_stmt st =
       |> List.concat_map (function
            | CasesAndBody (_, st) -> [ st ]
            | CaseEllipsis _ -> [])
-  | Try (_, st, xs, opt) -> (
+  | Try (_, st, xs, opt1, opt2) -> (
       [ st ]
       @ (xs |> Common.map Common2.thd3)
       @
-      match opt with
+      match opt1 with
       | None -> []
-      | Some (_, st) -> [ st ])
+      | Some (_, st) -> (
+          [ st ]
+          @
+          match opt2 with
```

Comment:
This doesn't look right, here `opt2` is only matched if `opt1` is `Some _`. If `opt1` is `None` and `opt2` is `Some _`, I think we still want to visit `opt2`, no? Shouldn't they be matched independently of each other?

Reply:
I think you're right. Thanks.